appveyor.yml

```diff
@@ -48,8 +48,16 @@ test:
 
 test_script:
   - dotnet restore
-  - IF "%CI_TYPE%" == "UNIT" dotnet test src\Cassandra.Tests -c Release -f %TARGET%
-  - IF "%CI_TYPE%" == "INTEGRATION" dotnet test src\Cassandra.IntegrationTests -c Release -f %TARGET% --where "cat=short"
+  - IF "%CI_TYPE%" == "UNIT" dotnet test src\Cassandra.Tests -c Release -f %TARGET% --result=TestResult.xml
+  - IF "%CI_TYPE%" == "INTEGRATION" dotnet test src\Cassandra.IntegrationTests -c Release -f %TARGET% --where "cat=short" --result=TestResult.xml
+
+after_test:
+  - ps: >-
+      if($true)
```

Comment:
its based on this documentation of appveyor

`To add multi-line PowerShell script covering an entire event:

on_success:
 - ps: |
 if($true)
 {
   Write-Host "Success"
 }`